android/engine/src/main/java/org/smartregister/fhircore/engine/domain/model/TaskStatus.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.domain.model
+
+enum class TaskStatus {
```

Comment:
is this not defined somehow through the status of the fhir resources, like is there a fhir file we can load that would include this?

Reply:
Yes. Forgot to delete this.